build/squire-raw.js

```diff
@@ -4117,6 +4149,23 @@ proto.setHTML = function ( html ) {
     // Fix cursor
     var node = frag;
     while ( node = getNextBlock( node, root ) ) {
+        // Replace the align attribute in P tag by style
+        if ( node.tagName.toUpperCase() === 'P' ) {
```

Comment:
This code is duplicated for both IMG and P elements. Can we just extract this into a separate function? Something like fixAlignmentAttributes.